utils/createFile.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs/promises');
+
+const createFile = (path, file) => fs.writeFile(path, file)
+
+module.exports = createFile
```

Comment:
remote this util and use native  fs.writeFile(path, file), beciuse is redundantly